apps/frontend/src/locales/en/translations.json

```
@@ -572,7 +572,11 @@
         "successMessage": "Your preferences have been saved.",
         "errorMessage": "Something went wrong, please try again.",
         "authErrorMessage": "Authentication failed, please try again.",
```

```suggestion
        "authErrorMessage": "Authentication failed",
```

Needs to be updated as per spec - I'm fine with current copy if it's approved by John (but we'll have to update D1 PR to match if so)